build/DependencyVersions.props

```diff
@@ -6,9 +6,9 @@
     <MsBuildPackagesVersion>15.1.548</MsBuildPackagesVersion>
     <DependencyModelVersion>1.0.3</DependencyModelVersion>
     <PlatformAbstractionsVersion>1.0.3</PlatformAbstractionsVersion>
-    <NuGetVersion>4.3.0-beta1-2418</NuGetVersion>
+    <NuGetVersion>4.3.0-preview1-2479</NuGetVersion>
     <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
-    <NewtonsoftJsonSideBySideVersion>6.0.4</NewtonsoftJsonSideBySideVersion>
+    <NewtonsoftJsonSideBySideVersion>9.0.1</NewtonsoftJsonSideBySideVersion>
```

Comment:
We can remove all of the SideBySide goop now that these versions are aligned. :)